Makefile.am

```diff
@@ -46,22 +46,26 @@ lib/ifupdown.sh: $(srcdir)/lib/ifupdown.sh.in
 sbindest=$(DESTDIR)$(sbindir)
 utilsdest=$(DESTDIR)$(utilsdir)
 etcdest=$(DESTDIR)$(sysconfdir)
+bashcompdest=$(etcdest)/bash_completion.d
 man5dest=$(DESTDIR)$(mandir)/man5
 man8dest=$(DESTDIR)$(mandir)/man8
 docdest=$(DESTDIR)$(docdir)
 
 install-data-hook:
 	install -m 755 bridge-stp $(sbindest)/bridge-stp
 	mkdir -pv $(utilsdest)
-	cp -rv lib/* $(utilsdest)
-	gzip -f $(utilsdest)/mstpctl.8
-	gzip -f $(utilsdest)/mstpctl-utils-interfaces.5
-	if [ -d $(etcdest)/network/if-pre-up.d ] ; then ln -sf $(utilsdest)/ifupdown.sh $(etcdest)/network/if-pre-up.d/mstpctl ; fi
-	if [ -d $(etcdest)/network/if-post-down.d ] ; then ln -sf $(utilsdest)/ifupdown.sh $(etcdest)/network/if-post-down.d/mstpctl ; fi
-	if [ -d $(etcdest)/bash_completion.d ] ; then ln -sf $(utilsdest)/bash_completion $(etcdest)/bash_completion.d/mstpctl ; fi
+	install -m 755 lib/ifupdown.sh $(utilsdest)
+	install -m 755 lib/mstpctl-utils.sh $(utilsdest)
+	install -m 755 lib/mstpctl_restart_config $(utilsdest)
+	if [ -d $(etcdest)/network/if-pre-up.d ] ; then ln -sf $(utilsdir)/ifupdown.sh $(etcdest)/network/if-pre-up.d/mstpctl ; fi
+	if [ -d $(etcdest)/network/if-post-down.d ] ; then ln -sf $(utilsdir)/ifupdown.sh $(etcdest)/network/if-post-down.d/mstpctl ; fi
```

Comment:
The reversion from utilsdest to utilsdir is actually intentional - the original use of $DESTDIR in the `ln` command is a bug ... I probably should have made this change a separate commit of its own to specifically identify it as a bug I was fixing...

In general, $DESTDIR is intended to facilitate the installation of the compiled software on a different system.  When compiling, $DESTDIR is not used, and any absolute path names (ld library paths, etc) that are compiled into the generated files do not include $DESTDIR.  When installing, $DESTDIR is used, but only to redirect the location of the installed files so that they can be packaged up and moved to another machine without actually installing them in real paths on the local machine; absolute paths compiled into the installed files should not be changed to include $DESTDIR.  After $DESTDIR is packaged and moved to the other machine, the idea is that it should be possible to extract the package into / on the target machine and everything will work as if `make install` (without DESTDIR) was run directly on the target machine.

If we use `ln -sf $(utilsdest)/ifupdown.sh $(etcdest)/network/if-pre-up.d/mstpctl` in the Makefile, then a valid symlink will be created under DESTDIR, but this will become a broken symlink after it is moved to the target machine: `/etc/network/if-pre-up.d/mstpctl -> /destdir/lib/mstpctl-utils/ifupdown.sh`.  We should instead use `ln -sf $(utilsdir)/ifupdown.sh $(etcdest)/network/if-pre-up.d/mstpctl` which creates a broken symlink on the local machine but causes a valid symlink to be installed on the target machine: `/etc/network/if-pre-up.d/mstpctl -> /lib/mstpctl-utils/ifupdown.sh`.